src/NuGet.Clients/NuGet.Console/Xamls/ConsoleContainer.xaml.cs

```diff
@@ -26,7 +34,14 @@ public ConsoleContainer()
                     await System.Threading.Tasks.Task.Run(
                         async () =>
                         {
```

Comment:
Is this Task.Run neccessary?

Reply:
That's from https://github.com/NuGet/NuGet.Client/pull/1729.  I think the intent was to run some initialization task (initially) on a background thread when VS is idle to improve VS startup time.  I haven't looked into whether there's a better alternative.  Do you have a concern or suggestion with the new code below running in this manner?

Reply:
No concern just seemed redundant and unnecessary